webapp/src/components/modals/channel_settings/channel_settings_filters.tsx

```diff
@@ -60,50 +62,22 @@ export default class ChannelSettingsFilters extends React.PureComponent<ChannelS
         this.props.onChange(newValues);
     };
 
-    getConflictingFields = (): {field: FilterField; issueTypes: IssueType[]}[] => {
```

Comment:
This function was moved to the utils file because it is used in two separate component files.